tests/Captioning/ConverterTest.php

```diff
@@ -16,23 +16,23 @@ public function testSubrip2WebvttConversion()
 
         $content = "WEBVTT
 
-00:00:00.000 --> 00:00:20.000
+00:00:01.000 --> 00:00:20.000
```

Comment:
Accordingly to current logic parse method checks result of regular expression execution.
First thing what we need to do - check current subtitle order ($subtitle[0] != $subtitleOrder++).
If we okay with order number then second what we want - validate timelines. The imposition of timelines is unacceptalbe.
And there is the problem. We do timeline validation through PHP DateTime objects comparison. And (i don't know the reason of this and don't founded any words about this on php.net) DateTime comparison refuse to compare milliseconds. I mean it's slill equals:
    $a = \DateTime::createFromFormat('H:i:s,u', '00:00:00,000');
    $b = \DateTime::createFromFormat('H:i:s,u', '00:00:00,999');
    print ($a >= $b); // will print 1 because condition statement returns true.

Now in context of converter test: we start validation from 00:00:00.000 and on first iteration in foreach (in parse method) after validation subtitle order we do comparise 00:00:00.000 and 00:00:00.000 - so validateTimelines will return false and validation will fail. Same result will be if first subtitle timeline will stars from lwer than 1 seconds. Because 00:00:00.000 and 00:00:00.999 still equal for PHP DateTime comparison.

I think what this problem can be resolved in next improvement (potentially need rewrine validateTimelines method logic to more complex than just comparing two DateTime objects).